docs/conf.py

```diff
@@ -104,12 +104,13 @@
     import sphinx_rtd_theme
     html_theme = 'sphinx_rtd_theme'
     html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
-    html_theme_options = {
-        'logo_only': True,
-    }
 except ImportError:
     html_theme = 'alabaster'
 
+html_theme_options = {
+    'logo_only': True,
```

Comment:
Specifying `logo_only` breaks the doc builds for me (running `cd docs; make html`):

    preparing documents... 
    Theme error:
    unsupported theme option 'logo_only' given

I think it needs to be in the block above, so that it's only specified for the `sphinx_rtd_theme`